cobbler/modules/managers/isc.py

```diff
@@ -305,14 +305,14 @@ def write_v6_config(self):
                 if distro is not None:
                     interface["distro"] = distro.to_dict()
 
-                if mac is None or mac == "":
+                if not mac or not ip_v6:
                     # can't write a DHCP entry for this system
+                    self.logger.warning("%s has no IPv6 or MAC address" % system.name)
```

Comment:
Found a lazy logger call. Or to be more precise Codacy did.

```suggestion
                    self.logger.warning("%s has no IPv6 or MAC address", system.name)
```